### ISSUE TITLE
Add Movie/TV shows sections

### ISSUE THREAD

**Issue by shijimasoft** [huynhsontung/Screenbox#486]
**Describe the solution you'd like**
Just like [Infuse (for iOS/macOS)](https://firecore.com/infuse), screenbox can group up movies and tv shows in different sections, and search metadata (for example: movie/tv show covers and preview, short descriptions etc.) for them using the tvdb/imdb (for movies and tv shows) and anidb (for anime).

**Describe alternatives you've considered**
Plex is a good alternative but still require a full server to download such metadata.

**Additional context**
All metadata should be saved offline and the last watched tv show episode (with timestamp) should be tracked

**Comment by United600** [comment]
Duplicate of #33

**Comment by heksesang** [comment]
> Duplicate of [#33](https://github.com/huynhsontung/Screenbox/issues/33)

#33 doesn't cover the grouping of movies and tv shows into different sections. So while this is partially dependent on the same feature as #33, I'd not say this is a duplicate?

**Comment by United600** [comment]
> [#33](https://github.com/huynhsontung/Screenbox/issues/33) doesn't cover the grouping of movies and tv shows into different sections. So while this is partially dependent on the same feature as [#33](https://github.com/huynhsontung/Screenbox/issues/33), I'd not say this is a duplicate?

You're right to say that it isn't exactly a duplicate but rather depends on it.
